app/uk/gov/hmrc/digitalservicestax/aa_data/package.scala

```diff
@@ -78,7 +78,7 @@ package object data extends SimpleJson {
 
   type IBAN = String @@ IBAN.Tag
   object IBAN extends RegexValidatedString(
-    """^[0-9]{8}$""", // TODO
+    """^[0-9]""", // TODO
```

Comment:
There must be some structure to IBAN's other than 'starts with a number', surely?

Reply:
I'm going to look into that today. Every country has a different sturcture apparently.